### DISCUSSION TITLE
Create a `.gitignore` file for the virtual environment

### DISCUSSION THREAD

**Discussion by brettcannon** [brettcannon/WWBD#28] [Ideas]
Effectively implement https://github.com/python/cpython/issues/83417 . Basically we would write a `.gitignore` file in the virtual environment's directory that just contained `*`.

**Comment by PaulFWatts** [comment]
Definitely a good idea imho.

**Reply by brettcannon** [reply]
Please make sure to upvote any ideas you like so that I know which ones have the most support. 🙂

**Comment by brettcannon** [comment]
Accepted as https://github.com/brettcannon/WWBD/issues/31 .
